src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractTestsVNext.cs

```diff
@@ -0,0 +1,85 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ServiceModel;
+using Infrastructure.Common;
+using WcfService;
+using Xunit;
+
+public static class MessageContractTestsVNext
```

Comment:
I think we should consider using the anticipated assembly version this will be a part of. Currently that is 4.3.0